Documentation/kube-flannel.yml

```diff
@@ -0,0 +1,74 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kube-flannel-cfg
+  namespace: kube-system
+  labels:
+    tier: node
+    app: flannel
+data:
+  cni-conf.json: |
```

Comment:
Where is it declared and by what consumed that this files gets written to _/etc/kube-flannel_?